<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Jayarees-MacBook-Pro.local" name="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice" tests="8" failures="0" timestamp="26 Nov 2019 12:43:16 GMT" time="62.491" errors="1">
  <testcase name="full_article_is_displayed_to_the_user" time="0.078" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.util.Objects.requireNonNull(Objects.java:203)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:85)
at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
at Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice.full_article_is_displayed_to_the_user(NEWSMO18_transaction_from_authorise_charge_notice.java:162)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- full_article_is_displayed_to_the_user -->
  <testcase name="i_am_a_user_of_Axate_and_I_am_on_the_registration_page_through_popbitch_FUN" time="14.176" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice"/>
  <testcase name="i_register_successfully_on_popbitch_with_one_pound" time="33.592" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice"/>
  <testcase name="opt_just_charge_me_on_the_finish_notice_and_click_ok" time="0.133" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice"/>
  <testcase name="click_ok_on_the_authorise_charge_notice" time="1.640" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice"/>
  <testcase name="twentyfivep_is_displayed_on_the_green_tab" time="0.558" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice"/>
  <testcase name="wallet_balance_is_detucted_by_twentyfivep" time="6.163" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice"/>
  <testcase name="free_point_is_detucted_by_twentyfivep" time="6.151" classname="Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice"/>
</testsuite> <!-- Stepdef.Newburylive.NEWSMO18_transaction_from_authorise_charge_notice -->
